docs/Ecosystem.md

```diff
@@ -134,6 +134,7 @@ Plugins maintained by the fastify team are listed under [Core](#core) while plug
 - [`fastify-sse`](https://github.com/lolo32/fastify-sse) to provide Server-Sent Events with `reply.sse( … )` to Fastify.
 - [`fastify-sse-v2`](https://github.com/nodefactoryio/fastify-sse-v2) to provide Server-Sent Events using Async Iterators (supports newer versions of Fastify).
 - [`fastify-tls-keygen`](https://gitlab.com/sebdeckers/fastify-tls-keygen) Automatically generate a browser-compatible, trusted, self-signed, localhost-only, TLS certificate.
+- [`fastify-tokenize`](https://github.com/Bowser65/fastify-tokenize) [Tokenize](https://github.com/Bowser65/Tokenize) plugin for Fastify that leverages the pain of managing authentication tokens, with built-in integration for `fastify-auth`
```

Comment:
```suggestion
- [`fastify-tokenize`](https://github.com/Bowser65/fastify-tokenize) [Tokenize](https://github.com/Bowser65/Tokenize) plugin for Fastify that leverages the pain of managing authentication tokens, with built-in integration for `fastify-auth`.
```

Reply:
@jsumners Are you sure `leverages` is the right word here?

Reply:
I've added the final dot and changed for a more adapted (and simpler) wording